index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>TouK recruitment task</title>
+</head>
+<body>
+    <div id="root"></div>
+    <script src="./out.js"></script>
```

Comment:
tego pliku tu nie powinno być w repo - powinien być w gitignore

Reply:
plik umieściłem tutaj ze względu na to, żeby mógłbyć podgląd na githubPages